### ISSUE TITLE
A bit more about cargo and crates.io

### ISSUE THREAD

**Issue by bugadani** [richardanaya/tour_of_rust#306]
Chapter 9 begins with mentioning sharing our code with others. The chapter also mentions crates.io, without anything about how to use other crates or cargo in general, also how to publish crates.

While this book is specifically about Rust and not necessarily about the wider ecosystem, I think it would be nice to at least include some pointers to these things.

**Comment by Zerotask** [comment]
@bugadani I totally agree with you. You basically always start a Rust project with either `cargo init` or `cargo new <project_name>` and then you use commands like `cargo run` or `cargo build`.

There is also an option to install additional cargo subcommands (cargo binaries) to improve your developer experience even further. I'd say cargo should be an own chapter.
